contracts/modules/TransferManager/MaximumVolumeTransferManager.sol

```diff
@@ -134,7 +134,6 @@ contract MaximumVolumeTransferManager is ITransferManager {
         uint256[] _endTimes, 
         uint256[] _rollingPeriodIntervals) 
     public withPerm(ADMIN) {
-        require(_maximumVolumes.length >= 1, "No restrictions provided");
```

Comment:
Put the sanity check back.
Add a test case with empty arrays to cover the branch.

Reply:
done!